book/src/installation-source.md

```diff
@@ -0,0 +1,80 @@
+# Installation: Build from Source
+
+Lighthouse builds on Linux, macOS, and Windows (via [WSL][] only).
+
+Compilation should be easy. In fact, if you already have Rust installed all you
+need is:
+
+- `git clone https://github.com/sigp/lighthouse.git`
+- `cd lighthouse`
+- `make`
+
+If this doesn't work or is not clear enough, see the [Detailed
+Instructions](#detailed-instructions) below. If you have further issues, see
+[Troubleshooting](#troubleshooting). If you'd prefer to use Docker, see the
+[Docker Guide](./docker.md).
+
+## Detailed Instructions
+
+1. Install Rust and Cargo with [rustup](https://rustup.rs/).
```

Comment:
While we're editing this, I think a remark about installing dependencies (CMake et al) is worthwhile. We could say something brief and then link to the relevant section below.